### PR TITLE
add `fmc_icmpv4_object`

### LINKED ISSUE #42: Add `fmc_icmpv4_object` resource and data source
API: /api/fmc_config/v1/domain/{domainUUID}/object/icmpv4objects

### PR DESCRIPTION
**Pull request by jabielecki**
Closes #42

No manual code.

### CODE REVIEW

File: gen/definitions/icmpv4_object.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: Icmpv4 Object
```
Comment (danischm) [inline gen/definitions/icmpv4_object.yaml:2]: `name: ICMPv4 Object`

File: gen/definitions/icmpv4_object.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/icmpv4objects
+data_source_name_query: true
+skip_minimum_test: true
```
Comment (danischm) [inline gen/definitions/icmpv4_object.yaml:5]: Is there a specific reason for this?

**Review by danischm — CHANGES_REQUESTED** [review]

File: gen/definitions/icmpv4_object.yaml (hunk 2 above)
Comment (jabielecki) [reply gen/definitions/icmpv4_object.yaml]: done, thanks

File: gen/definitions/icmpv4_object.yaml (hunk 1 above)
Comment (jabielecki) [reply gen/definitions/icmpv4_object.yaml]: done, thanks!

**Review by danischm — APPROVED** [review]

**Pull request merged**